cogito/get_test.go

```diff
@@ -94,25 +90,40 @@ func TestGetFailure(t *testing.T) {
 		},
 		{
 			name:    "concourse validation failure: empty version field",
-			source:  baseSource,
+			source:  baseGithubSource,
 			writer:  io.Discard,
 			wantErr: "get: empty 'version' field",
 		},
 		{
 			name:    "concourse validation failure: missing output directory",
-			source:  baseSource,
+			source:  baseGithubSource,
 			version: cogito.Version{Ref: "dummy"},
 			writer:  io.Discard,
 			wantErr: "get: arguments: missing output directory",
 		},
 		{
 			name:    "system write error",
 			args:    []string{"dummy-dir"},
-			source:  baseSource,
+			source:  baseGithubSource,
 			version: cogito.Version{Ref: "dummy"},
 			writer:  &testhelp.FailingWriter{},
 			wantErr: "get: preparing output: test write error",
 		},
+		{
+			name:    "missing gchat webhook",
+			source:  cogito.Source{Sinks: []string{"gchat"}},
+			version: cogito.Version{Ref: "dummy"},
+			writer:  &testhelp.FailingWriter{},
+			wantErr: "get: source: missing keys: gchat_webhook",
+		},
+		{
+			name: "validation failure: wrong sink key",
+			source: cogito.Source{
+				Sinks: []string{"slack", "gchat"},
+			},
+			writer:  io.Discard,
+			wantErr: "get: source: invalid sink: [slack]. Supported sinks: [gchat github]",
```

Comment:
same question as above in check tests. Any other name than slack?

Reply:
I changed, but be aware that you'll want to change the error message at every supported sinks change.